server/models/databaseLink.js

```diff
@@ -8,17 +9,17 @@ const seed = () => {
   const qry = database;
   client.query(qry, (err, result) => {
     if (err) {
-      console.log(err.toString());
+      winston.log(err.toString());
     } else {
-      console.log(result);
+      winston.log(result);
     }
   });
 };
 
 const connect = () => {
   client.connect()
     .then((err) => {
-      console.log('connection established');
+      winston.log('connection established');
```

Comment:
Make complete statements. What connection was established?

Does this log? Where does it log?